### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ckccameron**
Completes Iteration 1 setup for Futbol StatTracker.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'csv'
+require_relative 'games'
+require_relative 'teams'
+require_relative 'game_teams'
+
+class StatTracker
```
Comment (IamNorma) [inline lib/stat_tracker.rb:6]: Looks great so far. We'll add to stat_tracker as we go

File: lib/game_teams.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+              :team_id,
+              :hoa,
+              :result,
+              :settled_in,
+              :head_coach,
+              :goals,
+              :shots,
+              :tackles
+
+  def initialize(game_team_params)
```
Comment (dhalverson) [inline lib/game_teams.rb:13]: This looks good, we can double check we have initialized all the ones we need for the data next steps.

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+require 'csv'
+require_relative 'games'
+require_relative 'teams'
+require_relative 'game_teams'
+
```
Comment (dhalverson) [inline lib/stat_tracker.rb:5]: This require_relative should grow as we add new files for the project

File: lib/game_teams.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    @game_id = game_team_params[:game_id]
+    @team_id = game_team_params[:team_id]
+    @hoa = game_team_params[:hoa]
+    @result = game_team_params[:result]
+    @settled_in = game_team_params[:settled_in]
+    @head_coach = game_team_params[:head_coach]
+    @goals = game_team_params[:goals]
+    @shots = game_team_params[:shots]
+    @tackles = game_team_params[:tackles]
+  end
```
Comment (IamNorma) [inline lib/game_teams.rb:23]: We might be able to omit some of these. We can revisit after completing our stats methods

**Review by dhalverson — APPROVED** [review]
Looks good, I think we're in a good stage of our setup, and can be working on our methods soon.

File: test/games_test.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
+require './test/helper_test'
+require './lib/games'
+
+class GamesTest < Minitest::Test
```
Comment (IamNorma) [inline test/games_test.rb:4]: Test looks great

**Pull request merged**